src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Extensions/TimeSpanExtensions.cs

```diff
@@ -7,8 +7,10 @@ namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.Extensions
 {
     internal static class TimeSpanExtensions
     {
+        private const int MaxAttempts = 63;
+
         /// <summary>
-        /// This method calculates a random exponential backoff which lies between <see cref="RefreshConstants.DefaultMinBackoff"/> and <see cref="RefreshConstants.DefaultMaxBackoff"/>.
+        /// This method calculates randomized exponential backoff which lies between <see cref="RefreshConstants.DefaultMinBackoff"/> and <see cref="RefreshConstants.DefaultMaxBackoff"/>.
         /// </summary>
         /// <param name="interval">The maximum backoff to be used if <paramref name="interval"/> is less than <see cref="RefreshConstants.DefaultMaxBackoff"/>.
         /// If <paramref name="interval"/> is less than <see cref="RefreshConstants.DefaultMinBackoff"/>, <paramref name="interval"/> is returned.
```

Comment:
It's a bit odd to me that the self-contained extension has bounds defined by static type with reference to `Refresh`. If you ask me, bounds should be input parameters.

Reply:
Thats a good point. Right now we use this extension for calculating the backoff time during refresh only. But the idea of a backoff utility is pretty generic and should be reusable in other scenarios too. I can move the min/max to being input parameters.